tests/local_vespa/test_app/schemas/family_document.sd

```diff
@@ -201,53 +201,27 @@ schema family_document {
     rank-profile hybrid inherits default {
         inputs {
             query(query_embedding) tensor<float>(x[768])
+            query(description_bm25_weight) double: 1.0
+            query(description_closeness_weight) double: 1.0
         }
         function name_score() {
-            expression: attribute(name_weight) * bm25(family_name_index)
+            expression: bm25(family_name_index)
         }
         function description_score() {
-            expression: attribute(description_weight) * (bm25(family_description_index) + closeness(family_description_embedding))
+            expression: query(description_bm25_weight) * bm25(family_description_index) + query(descriptione_closeness_weight) * closeness(family_description_embedding)
         }
         first-phase {
-            expression: name_score() + description_score()
+            expression: (attribute(name_weight) * name_score()) + (attribute(description_weight) * description_score())
         }
         summary-features: name_score() description_score()
     }
     
-    rank-profile hybrid_no_description_embedding inherits default {
+    rank-profile hybrid_no_description_embedding inherits hybrid {
         inputs {
-            query(query_embedding) tensor<float>(x[768])
-        }
-        function name_score() {
-            expression: attribute(name_weight) * bm25(family_name_index)
+            query(description_closeness_weight) double: 0.0
```

Comment:
Getting my head around this, does setting this to 0.0 make it have no effect?

Reply:
yeah, it should do! description score just becomes `query(description_nativerank_weight) * nativeRank(family_description_index)`

Reply:
fyi i added a test for this in the latest few commits